src/components/Header.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import styled from 'styled-components';
+import { GITHUB_URL, LINKEDIN_URL, STACKOVERFLOW_URL } from 'utils/Urls';
+import heroLinkedin from './assets/linkedin.png';
+import heroGithub from './assets/github.png';
+import heroStackOverflow from './assets/stackoverflow.png';
+import profileImage from './assets/profile-pic.jpg';
+import heroImage from './assets/header-img.jpg';
+
+export const Header = () => {
+  return (
+    <HeaderContainer>
+
+      {/* Icons and links in the header */}
+      <HeaderIcons>
+        <a href={LINKEDIN_URL} target="_blank" rel="noreferrer">
+          <img src={heroLinkedin} alt="LinkedIn icon for the header" />
+        </a>
+        <a href={GITHUB_URL} target="_blank" rel="noreferrer">
+          <img src={heroGithub} alt="Github icon for the header" />
+        </a>
+        <a href={STACKOVERFLOW_URL} target="_blank" rel="noreferrer">
+          <img
+            src={heroStackOverflow}
+            alt="Stack Overflow icon for the header" />
+        </a>
+      </HeaderIcons>
+
+      {/* Title and name */}
+      <HeaderTitle>
+        <h3>Portfolio: Neah Rockstroh</h3>
+        <h1>Front-End</h1>
+        <h1>Developer</h1>
+      </HeaderTitle>
+
```

Comment:
Maybe this could use a text shadow to increase contrast towards the background, also the icons.